src/Components/Blueprints/BuildImagesButton.tsx

```diff
@@ -1,36 +1,138 @@
-import React from 'react';
+import React, {useState} from 'react';
 
-import { Button, ButtonProps } from '@patternfly/react-core';
+import {
+  ButtonProps,
+  Dropdown,
+  MenuToggle,
+  MenuToggleElement,
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuItem, Flex, FlexItem, Spinner, MenuToggleAction,
+} from '@patternfly/react-core';
 
 import { selectSelectedBlueprintId } from '../../store/BlueprintSlice';
 import { useAppSelector } from '../../store/hooks';
-import { useComposeBlueprintMutation } from '../../store/imageBuilderApi';
+import {
+  ImageTypes,
+  useComposeBlueprintMutation,
+  useGetBlueprintQuery,
+} from '../../store/imageBuilderApi';
+import {EditSaveButton} from "../CreateImageWizardV2/steps/Review/Footer/EditDropdown";
+import {skipToken} from "@reduxjs/toolkit/query";
 
-type BuildImagesButtonPropTypes = {
-  variant?: ButtonProps['variant'];
-  children?: React.ReactNode;
-};
-
-export const BuildImagesButton = ({
-  variant,
-  children,
-}: BuildImagesButtonPropTypes) => {
+export const BuildImagesButton = () => {
```

Comment:
This component is used in two places:

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/36f25648-8011-4e5f-8743-dd16f92b48a8)

When the props were changed a conflict between the attributes was introduced, causing this error:
![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/76a4aa8b-2ed4-4208-a993-3c18624b679b)

Originally the component was called with variant (which should be "primary" and "link" from top to bottom) and children.

There are two ways to resolve this, either we can try to bend the current logic to be reusable with the other "Build images" button or the link "Build images" button can be added independent of the original component since now it wouldn't be a code duplication as the buttons are quite different.

Reply:
This is how the second button should look like:
![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/aaa58c32-1022-4f30-8a39-06248fe538cd)